packaging/ubuntu-focal/debian/rules

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+
+# build with PIE hardening flag https://wiki.archlinux.org/index.php/Arch_package_guidelines/Security
+export GOFLAGS=-buildmode=pie
+
+%:
+	dh $@
+
+override_dh_installsystemd:
```

Comment:
Does the unit file for the socket need to be listed here too?

Reply:
I'm not completely sure, the socket is supposed to be named the same as its corresponding service file and this section tells it to install amazon-ecs-init.amazon-ecs-volume-plugin.service as amazon-ecs-volume-plugin.service. I think that would be enough for them to work together and for now it seems to activating correctly with the socket, but I will look in to whether the socket file name needs to change to fall under this too.